packages/crypto/src/lib/crypto.ts

```diff
@@ -452,3 +455,75 @@ export const checkSevSnpAttestation = async (
   // pass base64 encoded report to wasm wrapper
   return sevSnpVerify(report, data, signatures, challenge, vcekCert);
 };
+
+declare global {
+  // `var` is required for global hackery
+  // FIXME: `any` types for wasm are no bueno
+  // eslint-disable-next-line no-var, @typescript-eslint/no-explicit-any
+  var wasmExports: any;
+  // eslint-disable-next-line no-var, @typescript-eslint/no-explicit-any
+  var wasmECDSA: any;
```

Comment:
I don't think we sell need these defined on the global as in `v7` we scope the `wasm module`

Reply:
It is used [here](https://github.com/LIT-Protocol/js-sdk/blob/feature/lit-3579-v8-encryption-package-out-of-litnodeclient/packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts#L990) 🫠

Reply:
Hmm good catch! Making a task for me to go in and refactor this.